frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx

```diff
@@ -117,6 +117,14 @@ export default class LineAreaBarChart extends Component {
         t`Choose fields`,
       );
     }
+    const seriesOrder = (settings["graph.series_order"] || []).filter(
+      series => series.enabled,
+    );
+    if (dimensions.length > 1 && seriesOrder.length === 0) {
```

Comment:
When do we encounter this condition? I'm curious how it'll look like after throwing `ChartSettingsError`

Reply:
If you disable all columns, it will display the placeholder chart:
![image](https://user-images.githubusercontent.com/1328979/189640244-01c4b98b-63c3-4b2e-a788-5aa923ad75c0.png)

Reply:
Got it.